client/src/state/toasts/ToastsReducer.js

```diff
@@ -0,0 +1,23 @@
+import ACTION_TYPES from './ToastsActionTypes';
+
+const initState = [
+
+];
+
+function toastsReducer(state = initState, {type, payload}) {
+  switch (type) {
+    case ACTION_TYPES.DISPLAY:
+      return [...state, payload];
+
+    case ACTION_TYPES.DISMISS:
+      return state.map((toast) => (toast.key === payload.key ? {...toast, status: 'dismissing'} : toast));
```

Comment:
```suggestion
      return state.map((toast) => (toast.key === payload.key ? {...toast, dismissing: true} : toast));
```

This makes more sense, I think.